HidPkg/HidPkg.dec

```diff
@@ -35,6 +35,13 @@
   gHidKeyboardLayoutPackageGuid      = {0x57adcfa9, 0xc08a, 0x40b9, {0x86, 0x87, 0x65, 0xe2, 0xec, 0xe0, 0xe5, 0xe1}}
   gHidKeyboardLayoutKeyGuid          = {0xdcafaba8, 0xcde5, 0x40a1, {0x9a, 0xd3, 0x48, 0x76, 0x44, 0x71, 0x7e, 0x47}}
 
+[PcdsFixedAtBuild, PcdsPatchableInModule]
+  ## List of {InterfaceClass, InterfaceSubClass, InterfaceProtocol} tuples that should be excluded from being managed by
+  # the UsbHidDxe driver. The list is terminated by a record of all zeros.
+  # For example, to exclude USB keyboards, this would be set to {0x3, 0x1, 0x1, 0x0, 0x0, 0x0}

```

Comment:
The example is a good here, but would it be possible to add a link to either a file in the Tree or to a specification that enumerates the Interfaces, Interfaces subclasses and protocols?

Reply:
Revised comments to add spec reference and further description.

Reply:
Thank you!